src/gui/ResolveConflictsDialog.qml

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (C) 2023 by Matthieu Gallien <matthieu.gallien@nextcloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+import QtQml 2.15
+import QtQuick 2.15
+import QtQuick.Window 2.15 as QtWindow
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+import QtQml.Models 2.15
+import Style 1.0
+import com.nextcloud.desktopclient 1.0
+import "./tray"
+
+QtWindow.Window {
+    id: conflictsDialog
+
+    required property var allConflicts
+
+    flags: Qt.Window | Qt.Dialog
+    visible: true
+
+    width: 600
+    height: 800
+    minimumWidth: 600
+    minimumHeight: 800
+    title: qsTr('Solve sync conflicts')
+
+    onClosing: function(close) {
+        Systray.destroyDialog(self);
+        close.accepted = true
+    }
+
+    ColumnLayout {
+        anchors.fill: parent
+        anchors.leftMargin: 20
+        anchors.rightMargin: 20
+        anchors.bottomMargin: 20
+        anchors.topMargin: 20
+        spacing: 15
+        z: 2
+
+        EnforcedPlainTextLabel {
+            text: qsTr("%1 files in conflict").arg(delegateModel.count)
```

Comment:
@mgallien Will that work correctly with plurals/singulars https://doc.qt.io/qt-6/qobject.html#tr ?

Reply:
done